### PR TITLE
Fix Durham College

### PR DESCRIPTION
**Pull request by dl-eric**
Fixes Durham College's website and email

### CODE REVIEW

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -25290,13 +25290,13 @@
   },
   {
     "web_pages": [
-      "http://www.durhamc.on.ca/"
+      "https://durhamcollege.ca/"
     ],
     "name": "Durham College",
     "alpha_two_code": "CA",
     "state-province": null,
```
Comment (zsubzwary) [inline world_universities_and_domains.json]: It would be nice if you provide the state-province data...

Comment (dl-eric) [reply world_universities_and_domains.json]: Is this a new requirement now?

Comment (zsubzwary) [reply world_universities_and_domains.json]: It is just nice to have the information filled in.
If you know the info about Durham college you might as well know its state-province, so what is the harm in filling that info in?

**Comment by yigitguler** [comment]
Thank you @dl-eric. I am not sure if we should require state information for every university. However, especially for universities in the US, the state information can be important. Thanks for adding it.

**Pull request merged**